src/actor_api.c

```diff
@@ -2,8 +2,8 @@
 #include "effects.h"
 #include "battle/battle.h"
 
-extern s32 D_8029FBD0;
-extern s8 D_8029FBD4;
+extern s32 isGroupHeal;
```

Comment:
maybe IsGroupHeal since this is a global?

Reply:
Good catch. Annoyingly, its only used in a single function and has no business being a global. Alas.

Reply:
this and everything after it in the file can possibly be declared in-function static if we set it up correctly. it'd be good to do a pass of that on the whole game at some point (for symbols only used in one function)

Reply:
Agreed, I'd love to see that. Might help us find some "hidden" file splits.